### ISSUE TITLE
LekMap loads even if not chosen

### ISSUE THREAD

**Issue by Serpens66** [Viktor-Shcheglov/CivNQMod_2019#2]
Hi,
I dont have a reddit account, so I really dont know how to contact devs of lekmod/lekmap, so I will write here:

I downloaded the latest lekmap aand put it as instructed into the games maps folder.
But after that I hosted a multiplayer game without lekmod/lekmap, but the lekmap code was still executed, although I chose "Pangea". This resulted in different maps for me and the other players and we had a desync and I had to quit and remove the lekmap.
Is this intended?

**Comment by EnormousApplePie** [comment]
No, if you could provide a solution to that that would be wonderfull
